instant-xml-macros/src/lib.rs

```diff
@@ -1,19 +1,25 @@
 extern crate proc_macro;
 
+mod common;
```

Comment:
I'd prefer not to put this in a separate module for now, since `lib.rs` is still pretty small.

Reply:
done